.travis.yml

```diff
@@ -206,7 +215,6 @@ _helm_deploy: &helm_deploy
     --set global.alfrescoRegistryPullSecrets=quay-registry-secret \
     --wait \
     --timeout 20m0s \
-    --atomic \
```

Comment:
I know you removed this for testing but did you intend to leave it out of forget to add back?

Reply:
the cleanup is done at the end anyway, sadly with this atomic if the deployment fails initially it will get deleted on the spot even if keep env is set, so its better to remove it actually.